src/util/styles.ts

```diff
@@ -5,7 +5,12 @@ import { ICSSPixelUnitRule } from '@uifabric/merge-styles/lib/IRawStyleBase';
 const borderRadiusSoft: ICSSPixelUnitRule = '2px';
 const fontWeightBold: IFontWeight = '600';
 
+const fontSizeSmall = '1.2rem';
```

Comment:
Would you be alright with me tackling this in a separate PR?  Since Text and Block will be the primary consumers of these values, I'd like them to inform the types and shape we choose. 

I've started a spike here:
https://github.com/Microsoft/YamUI/pull/344